framework/src/postprocessors/SideIntegralVariablePostprocessor.C

```diff
@@ -36,17 +36,18 @@ SideIntegralVariablePostprocessor::SideIntegralVariablePostprocessor(
     _fv(_fv_variable)
 {
   addMooseVariableDependency(&mooseVariableField());
+
+  _qp_integration = !_fv;
+}
+
+Real
+SideIntegralVariablePostprocessor::computeFaceInfoIntegral(const FaceInfo * const fi)
+{
+  return MetaPhysicL::raw_value(_fv_variable->getBoundaryFaceValue(*fi));
```

Comment:
if we want it to work on internal faces we have to have an if here based on haveBlocks() then possibly take an internal face value.

one thing at a time is ok too!